### ISSUE TITLE
NASNet Mobile gives low accuracy on ImageNet

### ISSUE THREAD

**Issue by veronikayurchuk** [keras-team/keras#9674]
Hi! I am working on transferring NASNet Mobile weights from Keras to Pytorch.
After transferring the weights and making predictions of **the same image using both frameworks** I am getting absolutely **the same values** both in confidence and in any middle layers of the network.

However, the evaluation of the imagenet using transferred weights in Pytorch gave me
**Acc@1 38.110 Acc@5 60.844**

Also, I tested the script that I am using for evaluation on ImageNet and it works correct, as I got correct values for other architectures.

I was wondering if you could double check the weight that you provide for NASNet Mobile, probably there is some issue with them!

**Comment by ahundt** [comment]
This may be a legitimate issue, see https://github.com/keras-team/keras/issues/9674 as well

**Comment by taehoonlee** [comment]
The error has been fixed in #10209.
